docs/scripts_md/updateMRI_Upload.md

```diff
@@ -0,0 +1,52 @@
+# NAME
```

Comment:
rerun the pod2markdown when the changes are made

Reply:
Done